apps/crc_interactive.py

```diff
@@ -81,9 +81,15 @@ def _validate_arguments(self, args: Namespace) -> None:
             args: Parsed commandl ine arguments
         """
 
-        # Check wall time is between limits
-        if not self.min_time <= args.time <= self.max_time:
-            self.error(f'{args.time} is not in {self.min_time} <= time <= {self.max_time}... exiting')
+        # Check wall time is between limits, enable both %H:%M format as well as integer hours
+        check_time=0
+        if args.time.find(':')<0:
```

Comment:
The Python recommended operator for this type of conditional is `in`:

```python
if ':' in args.time:
    ...
```

However, since we are working with strings, I suggest `isdecimal` for a more robust approach:

```python
if args.time.isdeciaml():
    ...
```